src/bgen/Enums.cs

```diff
@@ -275,6 +275,27 @@ void GenerateEnum (Type type)
 			indent--;
 			print ("}");
 
+			if (BindingTouch.SupportsXmlDocumentation) {
+				print ($"/// <summary>Converts an array of <see cref=\"global::{type.FullName}\" /> enum values into an array of their corresponding constants.</summary>");
+				print ($"/// <param name=\"values\">The array of enum values to convert.</param>");
+			}
+			print ($"internal static NSString?[]? ToArray (this {type.Name}[]? values)");
+			print ("{");
+			indent++;
+			print ("if (values is null)");
+			print ("\treturn null;");
+			print ("var rv = new global::System.Collections.Generic.List<NSString?> ();");
+			print ("for (var i = 0; i < values.Length; i++) {");
+			indent++;
+			print ("var value = values [i];");
+			print ("rv.Add (value.GetConstant ());");
+			indent--;
+			print ("}");
+			print ("return rv.ToArray ();");
+			indent--;
+			print ("}");
+			print ("");
+
```

Comment:
Should we do this in a diff PR? If it is needed by other frameworks we want to keep it even if we rever the  changes for the CoreSpotlight framework. Would make more sense if we have a PR with the bgen change + a test and then the needed framework. This will mess our history.

Reply:
Yes, that sounds like a plan.

Reply:
This has been fixed now.